Sources/TuistKit/Commands/Cache/CacheWarmCommand.swift

```diff
@@ -17,13 +17,19 @@ struct CacheWarmCommand: ParsableCommand {
     )
     var path: String?
 
+    @Option(
+        name: .shortAndLong,
+        help: "The name of the flavor to be used when warming up the cache."
+    )
+    var flavor: String?
```

Comment:
As I mentioned earlier, I'd call this one `profile`.

Reply:
Done 👍